### ISSUE TITLE
User reported SUGGESTION 

### ISSUE THREAD

**Issue by abchugh** [slatex/sTeX-React#61]
A content SUGGESTION was logged by "a user" at the following url:

https://courses.voll-ki.fau.de/flash-cards/ai-1

## The issue as described by the user:
```
Give the content more space.
```

## The text highlighted while reporting this issue:
```
‍Γ,A⊢A
                                Ax
                              Γ⊢BΓ,A⊢B
                                weaken
                              ‍Γ⊢A∨¬A
                                TND
                              Γ⊢AΓ⊢BΓ⊢A∧B∧IΓ⊢A∧BΓ⊢A∧ElΓ⊢A∧BΓ⊢B∧ErΓ⊢AΓ⊢A∨B∨IlΓ⊢BΓ⊢A∨B∨IrΓ⊢A∨BΓ,A⊢CΓ,B⊢CΓ⊢C∨EΓ,A⊢BΓ⊢A⇒B⇒IΓ⊢A⇒BΓ⊢AΓ⊢B⇒EΓ,A⊢FΓ⊢¬A¬IΓ⊢¬¬AΓ⊢A¬EΓ⊢¬AΓ⊢AΓ⊢FFIΓ⊢FΓ⊢AFE
```

### The selected text was in the following section hierarchy:

1. GitLab: https://gl.mathhub.info//-/blob/main/source/<br/>FetchURL: /:sTeX/fragment?http://mathhub.info/MiKoMH/GenCS/pl0/slides/nd-sequents?pl0-nd-sequents?propSNDcalc

**Comment by abchugh** [comment]
The user here is referring to `propositional natural deduction calculus` card in `Propositional Reasoning` chapter. On small screens, we had to show a horizontal scrollbar.

**Comment by kohlhase** [comment]
yeah, I agree with the user, more horizontal space would be better in most cases. But I guess we will have to wait until HTML content is more reactive.

**Comment by Jazzpirate** [comment]
I'm pretty sure I can't do anything about that... I can't find the sources for precisely that block, and I have no idea what it should look like or what the problem precisely is, but looking at https://gl.mathhub.info/MiKoMH/GenCS/-/blob/main/source/pl0/slides/nd-sequents.en.tex, it looks to me like the "arrangement" of the formulae is "hardcoded" (if that makes sense). If more horizontal space is needed, the content will have to provide that and make room for that. MathML doesn't do line breaking, for pretty good reasons.

But also, I think 600px is an unreasonably small max-width for anything "top-level" already - even taking into account smart phones, I'm not sure I've seen a phone that is capable of running browsers and has a horizontal resolution lower than 1080...?

Either way, If you want to show formulae, squishing things even more just means that at some point, something will have to give, and if the latex sources put all of those inference rules in one row (=> one mathml row), then in one row they shall remain...

**Comment by abchugh** [comment]
> I'm pretty sure I can't do anything about that....MathML doesn't do line breaking, for pretty good reasons.

I suspected as much. A while ago I was also searching for some solutions for math line-breaking and found nothing useful. However, in this case there are 3 columns and it may or may not be possible to push one of them down on small screens.

> I think 600px is an unreasonably small max-width for anything "top-level"

I can't say for sure about Math content but 600px is actually the ideal size for reading usual content. It leads to around 65 characters per line for our font-size which fits most recommendations. See the screenshot in my previous comment. [some research](https://www.google.com/search?q=how+many+characters+per+line+for+readability). 

Also, I think there are very few cards (if any) that look bad because of the max-width constraint. Instead it's mainly because of the limited width of the screen.

>  I'm not sure I've seen a phone that is capable of running browsers and has a horizontal resolution lower than 1080...?

I think you are confusing length with width and also not considering the scaling factor used in phones which reduces the effective width. For eg. iPhoneXR has a screen resolution of 1792x828 but with a scaling factor of `2` it's effectively 896x414, i.e., a width of only 414px.

> I have no idea what it should look like or what the problem precisely is

Note the truncated 'Axiom' column on iPhone XR below:
![courses voll-ki fau de_flash-cards_ai-1(iPhone XR)](https://user-images.githubusercontent.com/5623836/216758897-3aaf03b3-9988-4cfd-b5cc-cf9074269dbd.png)
